package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cerebro",
   "productName": "cerebro",
-  "version": "0.6.0",
+  "version": "0.7.0",
```

Comment:
Next version should be 0.7.1

Reply:
Done https://github.com/cerebroapp/cerebro/pull/594/commits/ed45724c03a0dc2e0b368f99761ceee5ce901d7e